MOAS-LMS/Views/Course/_Module.cshtml

```diff
@@ -36,9 +36,15 @@
                     @Html.ActionLink("Delete", "Delete", "Module", new { id = Model.Id }, null) |
                     @Html.ActionLink("Create a new Activity", "Create", "Activity", new { id = Model.Id }, null)
                 </span>
+                <p></p>
```

Comment:
???

Reply:
det är för att "Edit | Delete | Create a new Activity" ska ha samma avstånd till "choose file"-knappen som "choose file"-knappen för Course (och Edit | Return to Course Overview, osv), och samma för Module. Utan <p>-taggarna är avståndet smalare.
![avstand](https://user-images.githubusercontent.com/38253015/43705871-16dabb66-9964-11e8-8727-776390fa94f0.png)

Reply:
Det där borde egentligen lösas med css.

Lägg till antingen `padding-top` eller `margin-top` på form-elementet